docs/use-github-in-bcgov/github-transition-guide.md

```diff
@@ -59,15 +60,30 @@ The important things to keep in mind are:
  
 If you don’t see information that could be helpful on this guide, please let us know through [our Jira Service Manager](https://citz-do.atlassian.net/servicedesk/customer/portal/2).
 
-## Technical information 
+## Technical Transition Guide
+
+Our technical transition guide helps GitHub users prepare for the upgrade, and confirms if users completed linking successfully. [Our IDIR transition guide](https://developer.gov.bc.ca/docs/default/component/bc-developer-guide/use-github-in-bcgov/github-transition-guide/) offers more details about IDIRs. 
+
+After **July 30th, 2024**, users cannot access the bcgov GitHub organization if they didn’t link their IDIR and GitHub ID. There are mechanisms in the system that force an IDIR login when you use your GitHub to login. 
 
-Our technical transition guide offers more details and helps GitHub users prepare for the upgrade.
+### Validating authentication
 
-The video shows the process, starting from the banner link on the bcgov GitHub organization, to the end:
+GitHub doesn't offer users a screen to confirm authentication. There are 3 alternative ways to validate linking GitHub IDs and IDIRS: 
```

Comment:
Does this contradict point 3?

Reply:
Or alternatively, perhasp the meaning is: 
```suggestion
GitHub doesn't display a confirmation when an account is linked to an IDIR . There are 3 alternative ways to validate linking GitHub IDs and IDIRS: 
```

Reply:
I'm deleting #2 because the banner is missable + other issues you've pointed out :) The other ways are more trustworthy.